src/wallet/wallet_args.h

```diff
@@ -44,8 +44,11 @@ namespace wallet_args
   concurrency. Log file and concurrency arguments are handled, along with basic
   global init for the wallet process.
 
-  \return The list of parsed options, iff there are no errors.*/
-  boost::optional<boost::program_options::variables_map> main(
+  \return
+    pair.first: The list of parsed options, iff there are no errors.
+    pair.second: Should the execution terminate due to --help or --version being specified
```

Comment:
I'd say "should the execution terminate succesfully", without referencing the current reasons since they just happen to be the current reasons.

Reply:
OK